config/locales/diaspora/en.yml

```diff
@@ -683,6 +683,7 @@ en:
         subject: "%{name} has mentioned you on diaspora*"
         limited_post: "You were mentioned in a limited post."
     private_message:
+        subject: "Diaspora: Private Message"
```

Comment:
That depends on your email client but I get your point. Does anyone have a better idea? The subjects of emails we send tend to be more verbose than “New private message” so to me it feels like that doesn't really fit.